phpunit.xml.dist

```diff
@@ -1,18 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-    bootstrap="vendor/autoload.php"
-    colors="true">
-    <coverage>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd" bootstrap="vendor/autoload.php" colors="true" convertDeprecationsToExceptions="true">
+    <coverage processUncoveredFiles="true">
         <include>
             <directory suffix=".php">./src</directory>
         </include>
     </coverage>
-
     <testsuites>
-        <testsuite name="laminas-oauth">
+        <testsuite name="laminas-authentication Test Suite">
```

Comment:
Please revert so that the only change is the addition of `convertDeprecationsToExceptions="true"`